akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -255,6 +255,12 @@ import akka.util.OptionVal
         failStage(ex)
       }
 
+      override def postStop(): Unit = {
+        if (!p.isCompleted) {
+          p.failure(new AbruptStageTerminationException(this))
```

Comment:
I love less lines of code to maintain, doesn't add any readability advantage for such one-liners